nodebuilder/das/config.go

```diff
@@ -23,6 +24,7 @@ func DefaultConfig(tp node.Type) Config {
 	switch tp {
 	case node.Light:
 		cfg.SampleTimeout = modp2p.BlockTime * time.Duration(cfg.ConcurrencyLimit)
+		cfg.SamplingWindow = light.Window.Duration()
```

Comment:
Do you wanna do the same for FN?

Reply:
We should set sampling window for FN/BN to infinity, otherwise it will be 0